swaybar/bar.c

```diff
@@ -180,6 +201,11 @@ static void wl_pointer_axis(void *data, struct wl_pointer *wl_pointer,
 		return;
 	}
 
+	if (check_bindings(bar, wl_axis_to_x11_button(axis, value),
+				WL_POINTER_BUTTON_STATE_PRESSED)) {
```

Comment:
This should send both a pressed event and released event. (at least, that seems to be the behaviour in i3)